### PR TITLE
Completed javascript-operators-and-expressions exercise.

### PR DESCRIPTION
**Pull request by Edward-J-Kim**
Screenshot 2023-05-08 at 3 28 05 PM

### CODE REVIEW

File: javascript-operators-and-expressions/main.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+console.log('Value of isAcidic:', isAcidic);
+console.log('typeof isAcidic:', typeof isAcidic);
+
+const headCount = 177;
+const isSparta = headCount === 300;
+console.log('Value of isSparta:', isSparta);
+console.log('typeof isSparta:', typeof isSparta);
+
+let motto = fullName;
+motto = fullName + ' is the GOAT';
```
Comment (shawnkost) [inline javascript-operators-and-expressions/main.js:37]: This should be using the `+=` operator

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change requested here Edward!

**Review by shawnkost — APPROVED** [review]
👍🏻

File: javascript-operators-and-expressions/main.js (hunk 1 above)
Comment (Edward-J-Kim) [reply javascript-operators-and-expressions/main.js]: Updated following operator!

**Pull request merged**